rhoas-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+### OIDC Clients for RHOAS
+quarkus.oidc-client.client-enabled=false
+# Red Hat SSO (refresh token read from "event-bridge.rhoas.sso.red-hat.refresh-token" property)
+quarkus.oidc-client.red-hat-sso.auth-server-url=${event-bridge.rhoas.sso.red-hat.auth-server-url}
+quarkus.oidc-client.red-hat-sso.client-id=cloud-services
+quarkus.oidc-client.red-hat-sso.credentials.secret=secret
+quarkus.oidc-client.red-hat-sso.grant.type=refresh
+# MAS SSO
+quarkus.oidc-client.mas-sso.auth-server-url=${event-bridge.rhoas.sso.mas.auth-server-url}
```

Comment:
@kostola Can you make the ENV variable naming consistent with what we already have please. Current approach is UPPERCASE with words separated by `_` e.g `RHOAS_SSO_MAS_AUTH_SERVER_URL`

Take a look at `manager/src/main/resources/application.properties` for examples 👍

Reply:
This is already working thanks to [MicroProfile Config mapping between env vars and properties](https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#environment-variables-mapping-rules).
Since the mapping is only from env vars to properties and not backwards, I preferred this syntax because it allows both methods to be used, while explicitly using env vars name prevents properties to be used.
@robpblake what do you think?

Reply:
@kostola Thank you for the link. That's useful to know and your suggestion is the right way IMHO.

When we make decisions like this, it's important that we're consistent across the project. Making the change in a PR only for a small subset of the properties and then not communicating it as an implementation choice to the rest of the team means we end up in a position where we have multiple approaches to the same thing and people won't follow the same approach as you in the future. I've put a proposal in our Google Chat room about how we capture these decisions.

In the meantime, could you please create another JIRA task to make all properties follow this approach.

Reply:
I created the JIRA task: https://issues.redhat.com/browse/MGDOBR-215